src/locales/en-gb.d.ts

```diff
@@ -0,0 +1,96 @@
+interface enGB {
+  locale: () => string;
+  actions: {
+    edit: () => string;
+    delete: () => string;
+    reset: () => string;
+  };
+  actionPopover: {
+    ariaLabel: () => string;
+  };
+  actionToolbar: {
+    selected: () => string;
+  };
+  batchSelection: {
+    selected: (count: number | string) => string;
+  };
+  confirm: {
+    no: () => string;
+    yes: () => string;
+  };
+  date: {
+    formats: {
+      inputs: () => string[];
+      javascript: () => string;
+    };
+  };
+  errors: {
+    messages: {
+      formSummary: (
+        errors: number | string,
```

Comment:
`errors` and `warnings` should only be strings because they're passed into `isSingular` which passes them into `parseInt`

![image](https://user-images.githubusercontent.com/14963680/134190899-8f4145a6-08f6-4e33-8bd8-c301c868ff71.png)

Reply:
Here based on the prop types should be a number type.
![Zrzut ekranu 2021-09-22 o 10 24 35](https://user-images.githubusercontent.com/77274590/134309039-e7dd9d3f-ba64-4cae-90ac-227d8ab07836.png)

Reply:
Ok in that case, the `isSingular` function needs to change if the `count ` parameter is a number - because it won't be valid in TypeScript if we try to pass a number into `parseInt` as the first parameter.

So the simple solution would be to check the type of `count` in `isSingular`, and if it's not a string don't pass it into `parseInt`. I.e something like this: 

```js
export const isSingular = (count) =>
  typeof count === "string" ? parseInt(count, 10) === 1 : count === 1;
```